### DISCUSSION TITLE
Modify URI

### DISCUSSION THREAD

**Discussion by MhdTarhini** [filamentphp/filament#14309] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

Can we append record name to url for visibility. For an instance Disney is https://exmaple.com/services/1  but would be great if we could also use https://exmaple.com/services/disney so it’s visible when sharing urls etc.

**Comment by leandrocfe** [comment, marked as answer]
You could use a `slug`. Override the getRouteKeyName method on the Eloquent model:

```php
public function getRouteKeyName(): string
{
    return 'slug';
}
```
